rfhub2/cli/keywords_importer.py

```diff
@@ -185,67 +202,78 @@ def delete_outdated_collections(
             self.client.delete_collection(collection)
         return collections_to_delete
 
-    def add_collections(self, collections: List[Dict]) -> List[Dict[str, int]]:
+    def add_collections(
+        self, collections: Tuple[List[CollectionUpdate], List[KeywordUpdate]]
```

Comment:
I need to check this annotation as well. BTW, is there some kind of type validator? Maybe it could be added to CI?

Reply:
We can investigate this. In the meantime, PyCharm is warning about such discrepancies between annotation and code pretty well

Reply:
@MaciejWiczk Once we merge our PRs I will work on adding mypy check and resolving any typing issues. There are typing stubs for SQLAlchemy that will hopefully work. As far as I know there are none for robot framework so part of the code will be unchecked. But still I really like this idea and want to introduce it.

Reply:
>  Once we merge our PRs I will work on adding mypy check and resolving any typing issues.

Good, I feel like I could made a few mistakes here and there.

> There are typing stubs for SQLAlchemy that will hopefully work.

You mean to make MyPy not go crazy with SQLAlchemy?

> As far as I know there are none for robot framework so part of the code will be unchecked.

You mean there is no stubs for RFWK? Maybe we could exclude `*.robot, *.resource`  files alltogether?

Reply:
MyPy is based on type annotations which are still not available for some Python packages, like SQLAlchemy or Robot Framework. For SQLAlchemy there is external package with stubs maintained by Dropbox, but for Robot Framework there is nothing yet. So any code in our project that interacts with imports from robot package won't be properly checked.